app/viewmodels/checkAnswers/income/NotTaxableAmountSummary.scala

```diff
@@ -14,29 +14,29 @@
  * limitations under the License.
  */
 
-package viewmodels.checkAnswers
+package viewmodels.checkAnswers.income
 
 import controllers.journeys.income.routes.NotTaxableAmountController
 import models.{CheckMode, UserAnswers}
 import pages.income.NotTaxableAmountPage
 import play.api.i18n.Messages
+import uk.gov.hmrc.govukfrontend.views.Aliases.{Key, Value}
 import uk.gov.hmrc.govukfrontend.views.viewmodels.summarylist.SummaryListRow
 import viewmodels.govuk.summarylist._
 import viewmodels.implicits._
 
 object NotTaxableAmountSummary {
 
-  def row(answers: UserAnswers, taxYear: Int, businessId: String)(implicit messages: Messages): Option[SummaryListRow] =
-    answers.get(NotTaxableAmountPage).map {
-      answer =>
-
-        SummaryListRowViewModel(
-          key = "notTaxableAmount.checkYourAnswersLabel",
-          value = ValueViewModel(answer.toString),
-          actions = Seq(
-            ActionItemViewModel("site.change", NotTaxableAmountController.onPageLoad(taxYear, businessId, CheckMode).url)
-              .withVisuallyHiddenText(messages("notTaxableAmount.change.hidden"))
-          )
+  def row(answers: UserAnswers, taxYear: Int, authUserType: String, businessId: String)(implicit messages: Messages): Option[SummaryListRow] =
+    answers.get(NotTaxableAmountPage, Some(businessId)).map { answer =>
+      SummaryListRowViewModel(
+        key = Key(content = s"notTaxableAmount.checkYourAnswersLabel.$authUserType", classes = "govuk-!-width-two-thirds"),
+        value = Value(content = s"£${answer.setScale(2)}", classes = "govuk-!-width-one-third"),
```

Comment:
These need comma formatting e.g. '1000' -> '1,000'. If they also need decimal place formatting you can use the SelfEmploymentService. convertBigDecimalToMoneyString method

Reply:
Have spoken with Chris, 2DP is correct for the CYA page so only the commas are needed